content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-client-extensions-model.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Add custom methods to your models (Prisma Client extensions)'
+metaTitle: 'Prisma Client extensions: model (Preview)'
+metaDescription: 'Extend the functionality of Prisma Client'
+tocDepth: 3
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+This page is in draft.
+
+</Admonition>
+
+From version **TBD**, you can use `model` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) to add custom methods to your models.
+
+To create Prisma Client extensions, use the [`$extends`](/concepts/components/prisma-client/client-extensions#about-prisma-client-extensions) client-level method. To add custom methods to your models, use the `model` extension type.
+
+Possible uses include the following:
+
+- New operations to operate alongside existing Prisma Client operations, such as `findMany`
+- Encapsulated business logic
+- Repetitive operations
+- Model-specific utilities
+
+You can add methods to a specific model in your schema or to all models in your schema.
```

Comment:
Current order seems weird to me:
```suggestion
You can add methods to a specific model in your schema or to all models in your schema.

Possible uses include the following:

- New operations to operate alongside existing Prisma Client operations, such as `findMany`
- Encapsulated business logic
- Repetitive operations
- Model-specific utilities
```

Reply:
There was a logic to this sequence, but looking again with fresh eyes, I think I can roll that sentence into one of the intro sentences. This will also make the intro less repetitive.